### PR TITLE
Fix RSPEC templating files to handle the multiple diagnostic IDs

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

File: sonaranalyzer-dotnet/build/rspec-templates/CSharpTestTemplate.cs [hunk 1]
```diff
@@ -30,7 +30,8 @@ public class $DiagnosticClassName$Test
         [TestCategory("Rule")]
         public void $DiagnosticClassName$()
         {
-            Verifier.VerifyAnalyzer(@"TestCases\$DiagnosticClassName$.cs", new $DiagnosticClassName$());
+            Verifier.VerifyAnalyzer(@"TestCases\$DiagnosticClassName$.cs",
```
Comment (valhristov) [inline sonaranalyzer-dotnet/build/rspec-templates/CSharpTestTemplate.cs]: Why is this required? Should we create a policy of adding all arguments with index>=1 on the next line?

Comment (Evangelink) [inline sonaranalyzer-dotnet/build/rspec-templates/CSharpTestTemplate.cs]: It is just that lately I often ended-up with long names so I had to put them on the line.
